.github/renovate.json5

```
@@ -15,6 +15,8 @@
   ],
   // Add PR footer with empty release note by default.
   prFooter: '**Release note**:\n```other dependency\nNONE\n```',
+  // Only rebase when there are conflicts. Prow tests against the latest state of master branch anyway before merging the PR.
+  rebaseWhen: "conflicted",
```

Nit: To keep the formatting consistent
```suggestion
  rebaseWhen: 'conflicted',
```